65black/StackOverflow-lite
# #160381417 Add Comment to Answer

## What does this PR do?
Adds logic for the` POST /api/v1/questions/:questionId/answers/:answerId/comments` endpoint.
The user must be signed in to access this endpoint.

If the user is signed in, then hitting this endpoint will post a comment to a previously existing answer.

### Description of Task to be completed?
Have the following endpoints working:

> POST /api/v1/questions/:questionId/answers/:answerId/comments

### How should this be manually tested?
After cloning the repo, cd into it and run `npm run start-dev`. Follow these steps:

 - Open postman and hit the login endpoint first to get your auth token.
 - Hit the above endpoint making sure to replace the IDs with an existing question and answer.
 - You should get a response of the comment you posted.

### Screenshots
![scr](https://user-images.githubusercontent.com/36507972/45308206-d9a34a80-b518-11e8-8b41-ecf607d32b09.JPG)

### What are the relevant pivotal tracker stories?
#160381417
